src/test/regress/expected/failure_ddl.out

```diff
@@ -69,9 +69,7 @@ SELECT citus.mitmproxy('conn.onQuery(query="^BEGIN TRANSACTION ISOLATION LEVEL R
 (1 row)
 
 ALTER TABLE test_table ADD COLUMN new_column INT;
-ERROR:  connection to the remote node localhost:xxxxx failed with the following error: server closed the connection unexpectedly
-	This probably means the server terminated abnormally
-	before or while processing the request.
+ERROR:  failure on connection marked as essential: localhost:xxxxx
```

Comment:
Well, now that client_error_messages here is `ERROR`, we lose the previous error message. If you change it to `WARNING` do you see old errors as warnings here? If so, I'd be nice to get those back